pom.xml

```diff
@@ -1511,11 +1511,36 @@
       </activation>
     </profile>
 
+    <profile>
+      <id>spark3.1.x</id>
```

Comment:
just spark3.1?

Reply:
using spark3.1 is ok, but i prefer to keep consistent with `spark3.0.x`.